src/Mod/Arch/archviewproviders/view_reinforcement_linear.py

```diff
@@ -0,0 +1,39 @@
+# ***************************************************************************
+# *   Copyright (c) 2020 Bernd Hahnebach <bernd@bimstatik.org>              *
+# *                                                                         *
+# *   This program is free software; you can redistribute it and/or modify  *
+# *   it under the terms of the GNU Lesser General Public License (LGPL)    *
+# *   as published by the Free Software Foundation; either version 2 of     *
+# *   the License, or (at your option) any later version.                   *
+# *   for detail see the LICENCE text file.                                 *
+# *                                                                         *
+# *   This program is distributed in the hope that it will be useful,       *
+# *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+# *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+# *   GNU Library General Public License for more details.                  *
+# *                                                                         *
+# *   You should have received a copy of the GNU Library General Public     *
+# *   License along with this program; if not, write to the Free Software   *
+# *   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  *
+# *   USA                                                                   *
+# *                                                                         *
+# ***************************************************************************
+
+__title__ = "FreeCAD linear reinforcement view object"
+__author__ = "Bernd Hahnebach"
+__url__ = "http://www.freecadweb.org"
+
+import Arch_rc
+
+from .view_reinforcement_generic import ViewProviderReinforcementGeneric
+
+
+False if Arch_rc.__name__ else True  # dummy usage
+
+
+class ViewProviderReinforcementLinear(ViewProviderReinforcementGeneric):
+
+    def getIcon(
+        self
+    ):
```

Comment:
Are you kidding with this style? What is so hard about regular style?
```py
def getIcon(self):
    return
```

Reply:
Stick to facts!
it is just the way I do it on my own code. This rebar2 has been lying around for years. It started with an own hack, and on one point I thought it may makes sense to share it. I have no problem with changing.